docs/docs/general/01-Introduction/why-askui.md

```diff
@@ -1,21 +0,0 @@
----
```

Comment:
Was the removal of this file intentional? Can you shortly explain the reasoning to me?

Reply:
Yes, with pleasure 😃 

We noticed it takes too long for a user to actually get started. As we are marketing to developers they are usually not interested in the WHY of our product but more in the WHAT CAN IT DO FOR ME question. So by removing the why-askui and the homepage we hope to get them to their first Hello World faster.